resources/icons/dropdown_down.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+  <path transform="rotate(180 12 12)" fill="" d="M12.1872807,7.00692544 C12.4141917,7.02726144 12.6351377,7.12378244 12.8065087,7.29515444 L19.8740787,14.3627244 C20.2655697,14.7542144 20.2690747,15.3854394 19.8758277,15.7786864 C19.4853037,16.1692114 18.8464497,16.1635214 18.4598657,15.7769374 L12.0831877,9.40026044 L5.70651067,15.7769374 C5.31502067,16.1684274 4.68379467,16.1719334 4.29054767,15.7786864 C3.90002367,15.3881624 3.90571267,14.7493084 4.29229667,14.3627244 L11.3598667,7.29515444 C11.5862877,7.06873444 11.8928977,6.97209244 12.1872807,7.00692544 L12.1872807,7.00692544 Z"/>
```

Comment:
you can use dropdown_up and use transform for container https://github.com/status-im/status-react/blob/d834b6e37d1ecc52a2367676cee0b338c383087f/src/status_im/ui/components/tooltip/views.cljs#L29